tests/org.eclipse.n4js.spec.tests/xpect-tests/Ch05_04__Redefinition_of_Members/Req70_OverrideFieldFinal.n4js.xt

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
This is not a review comment but general observation. It is interesting that as opposed to N4JS, if we redeclare a field in the subclass, a new field is created and this new field _shadows_ that field in the superclass.

```java
// JAVA CODE!
class A {
	public String field;

	public void foo() {
		field = "Something else";
	}
}

interface I {
	int field2 = 10;
}

class B extends A implements I {
	public String field = "hello";
	final int field2 = 42;
}

class C extends B {

	String field = "hello";
	int field2 = 42;
}

public class Test {
	public static void main(String[] args) {
		B b = new B();
		b.foo();
		System.out.println(b.field);               // Output: hello
		System.out.println(((A) b).field);     //  Output: Something else
	}
}
```